### ISSUE TITLE
Deletion -- 620, 630 Asynchronous Program Presentations & Views & Revisions to 600-618 Synchronous Programs & Attendance

### ISSUE THREAD

**Issue by KathleenSullivan** [IMLS/public-libraries-survey#69]
**Name**: Kathleen Sullivan 

**State/affiliation**: Washington State Library

**Description of change _(include existing data element definitions)_**:

- Delete 620, 630 to eliminate distinction between synchronous and recorded/asynchronous virtual program content and attendance

**620 Total Number of Asynchronous Program Presentations (TOTPRES)**
_An asynchronous program presentation is any recording of program content that cannot be viewed live as it unfolds (i.e., on-demand streaming). Only include program presentations posted during the reporting period. Regardless of the number of platforms on which a presentation is posted, count each unique presentation only once. Include program sessions hosted on Facebook Premiere that are not facilitated by a staff member. Count asynchronous program presentations at the administrative entity level; do not duplicate numbers at each branch. Include recordings of synchronous program sessions that were available for asynchronous viewing after the session ended._

**630 Total Views of Asynchronous Program Presentations within 30 Days (TOTVIEWS)** 
The count of views of asynchronous program presentations for a period of thirty (30) days after the presentation was posted, even if that period extends beyond the survey reporting period (or fiscal year). For program presentations made available via Facebook, count unique 1-minute views of each video. For those made available via other platforms, count unique views of each video. For program presentations that are recordings of synchronous virtual program sessions, exclude synchronous attendance; these should be counted under Synchronous In-Person Onsite Program Attendance, Synchronous In-Person Offsite Program Attendance, or Synchronous Virtual Program Attendance (data elements 616, 617, or 618). 

- Revise title and definitions of elements 600-618 to eliminate the word  _synchronous_ and instead refer simply to _programs_.

[I'll avoid pasting in pages of elements for this one redaction, but I don't think more significant revisions to the text are required.]

- Revise element 618 to incorporate views of recordings for up to 30 days

[existing element]
**618 Synchronous Virtual Program Attendance (VIRATTEN)**
_The count of live attendance at virtual program sessions. Regardless of the number of formats in which a program session is offered, each attendee or view should only be counted once. Each attendee should be counted in the format category in which they attended or viewed the program session. Count each participant device connected to a virtual program as a single attendee. For program sessions hosted on Facebook Live, YouTube Live, or similar platforms, count peak concurrent viewers. For those hosted on videoconferencing platforms, count the maximum number of non-staff participants during the session. For virtual program sessions that are also recorded for later, on-demand, asynchronous viewing, exclude views that occur after the session has ended; these should be counted under Total Views of Asynchronous Program Presentations (data element 630). For program sessions that also have an in-person component, exclude in-person attendance; this should be counted under Synchronous In- Person Onsite Program Attendance or Synchronous In- Person Offsite Program Attendance (data elements 616 or 617). This figure is a subset of the Total Attendance at Synchronous Programs (data element 610). See Total Attendance at Synchronous Programs for more information about counting program session attendance._ 

[proposed revision to fold in views of recordings]
**REVISED 618 Total Virtual Program Attendance (VIRATTEN)**
_The count of views or plays of virtual program sessions. Regardless of the number of formats in which a program session is offered, each attendee access should only be counted once. Each attendee should be counted in the format category in which they attended, viewed, listened to or otherwise accessed the program session. Count each participant device connected to a virtual program as a single attendee. Count the views, plays or other access of program presentations for a period of thirty (30) days from the live program or posting of the presentation (whichever is earlier), even if the 30-day period extends beyond the survey reporting period (or fiscal year). For program presentations made available via Facebook, count unique 1-minute views of each video. For those made available via other platforms, count unique views, plays or other access of each virtual program. For those hosted on videoconferencing platforms, count the maximum number of non-staff participants during the session. For program sessions that also have an in-person component, exclude in-person attendance; this should be counted under In-Person Onsite Program Attendance or In-Person Offsite Program Attendance (data elements 616 or 617). This figure is a subset of the Total Attendance at Programs (data element 610). See Total Attendance at Programs for more information about counting program session attendance._ 

**Justification for deletion of existing data element**:

Multiple public library systems from Washington state requested this refinement. Distinguishing live views/listens and later access of recordings is burdensome -- or sometimes impossible, depending on the platform used. Libraries also didn't find it particularly valuable to track or compare whether patrons virtually attended a program live or later. Asynchronous attendance is simply assumed as a feature of virtual programs and attendance. 

**If known, potential methodological issues related to the change (e.g., data unreliability/bias, potential data collection challenges)**:
If anything, this would simplify collection of virtual attendance figures and head off unreliable guessing about where live attendance ended and views/listens of recordings began. ...It should also not disrupt longitudinal comparisons, since the revised Total Virtual Attendance could be compared to the sum of Synchronous Virtual Program Attendance and Total Views of Asynchronous Program Presentations within 30 Days (starting in 2022, when the 30-day counting period began).

**Comment by enielsen-air** [comment]
I am frankly surprised it took this long for someone to propose deleting TOTPRES and TOTVIEWS!

Before I offer my comments, I need to point out that the FY24 PLS changes ballot that states just voted on included revisions to both of these items, and the revisions above don't reflect those changes (since the ballot results have not yet been announced).

That said, for whatever my opinion is worth, I disagree with incorporating the recorded programs and "plays" counts into the other program definitions for a few reasons:
1. We had much discussion back in 2020 about how recordings do not meet the definition of a program that occurs with a group of people, at a specific time, to allow interaction between those attendees and library staff (or another facilitator).
2. Revising the attendance counts to include the "plays within 30 days" would do very little to ease the reporting burden to capture these audience metrics at just the right moment.
3. My impression (which could be biased) is that "recorded programs" were really an anomaly of the depths of the pandemic, and that many of the libraries that may have been doing them in 2020 and early 2021 have stopped. (I haven't looked at the FY22 data yet, so this impression is based mostly on the cognitive interviews we did this spring about the revisions on the FY24 ballot.)

In sum, I think deleting 620 and (especially) 630 is a good idea, but it doesn't make sense to revise 618.

**Comment by VTLIB1** [comment]
Apologies, Kathleen - I just created a separate issue to suggest deleting 620 & 630 (#71). From where I am, I'd be in favor of dropping the questions entirely at the Federal level.

**Comment by BAAugspurger** [comment]
Good morning - new to SDC position in KY so getting up to speed on this issue.  I was approached about the uselessness of 30-day timeframe viewing counts. Their suggestion was to make reporting Asynchronous viewing counts for the FY cycle. Food for thought :-)

**Comment by enielsen-air** [comment]
I'm closing this issue as the LSWG decided in Fall 2023 not to advance this proposal for consideration.
